dstk-infra/apollo/src/graphql/model-version/modelVersion.ts

```diff
@@ -3,7 +3,7 @@ import { MLModel, ObjectionMLModel } from '../model/model.js';
 import { Model } from 'objection';
 
 export const MLModelVersion = objectType({
```

Comment:
Do we need to add in `t.boolean('isFinalized')?

Reply:
Good call, I wasn't going to expose that through the API but you'll def need it for determining UI state. I'll add it in!